data/commodities.txt

```diff
@@ -112,43 +117,58 @@ trade
 		"linen sheets"
 		"mannequins"
 		"men's shoes"
+		"parkas"
 		"protective clothing"
 		"rayon"
 		"running shoes"
 		"sailcloth"
 		"socks"
 		"surgical scrubs"
+		"swimsuits"
 		"textiles"
 		"umbrellas"
 		"underwear"
 		"uniforms"
 		"winter coats"
 		"women's shoes"
+		"waterproof fiber"
 		"wool"
 		"work boots"
 	commodity "Metal" 190 590
 		"aluminized steel"
 		"aluminum"
+		"beryllium"
```

Comment:
The "metals" section mostly consists of metallic elements already, so I'll leave it up to MZ if he wants to go in that direction.